test/integration/expression-tests/let/zoom/test.json

```diff
@@ -23,22 +23,6 @@
       "isZoomConstant": false,
       "type": "number"
     },
-    "outputs": [20],
```

Comment:
Why did the whole "serialized" section disappear here (and in zoom/basic)?

Reply:
Oh, I thought `serialized` is only needed when it differs from `expression`. I can add it back.

Reply:
That's a good idea! But no it doesn't do that (yet)...